src/app/containers/Timestamp/index.jsx

```diff
@@ -29,22 +32,60 @@ const formatTimestamp = dateObj => {
   return `${dayNumeric} ${monthLong} ${fullYear}`;
 };
 
-const TimestampContainer = ({ timestamp }) => {
-  const dateObj = new Date(timestamp);
+const isTenHoursAgo = milliseconds => {
+  const now = Date.now();
+  return now - milliseconds >= 10 * 60 * 60 * 1000;
+};
+
+const timestampWithPrefixUpdated = (datetime, milliseconds) => (
+  <Timestamp datetime={datetime} prefix="Updated">
+    {milliseconds}
```

Comment:
Is this `milliseconds`?

Reply:
it's a unix timestamp but I didn't want to use the word `timestamp` to avoid confusion

Reply:
I dont think it is though, it either gets `formatTimestamp()` or `relativeTime()`

Reply:
sure, I'll figure out a new name for it